templates/nsis/boringInstaller.nsh

```diff
@@ -7,13 +7,28 @@
 
 !ifndef BUILD_UNINSTALLER
   Function StartApp
-    ${StdUtils.ExecShellAsUser} $0 "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "open" ""
+    ${GetParameters} $R0
+    ${GetOptions} $R0 "--update" $R1
+    ${IfNot} ${Errors}
+      ${StdUtils.ExecShellAsUser} $0 "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "open" "--updated"
+    ${Else}
+      ${StdUtils.ExecShellAsUser} $0 "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "open" ""
+    ${endif}
+  FunctionEnd
+
+  Function licensePre
```

Comment:
Error: resolving install function "licensePre" in function "mui.LicensePre_22.5.19"
 it doesn't work any more, do I need to do something like export or change visiblity?